translation.rst

```diff
@@ -1104,6 +1104,13 @@ just to run some code. Imagine a console command that renders Twig templates
 of emails in different languages. You need to change the locale only to
 render those templates.
 
+.. note::
+
+    The LocaleSwitcher will apply the locale at a request level
+    this means that it will be available only for that request. A
+    redirect, for example, will cancel the LocaleSwitcher's effect.
+    For a permanent locale switch between requests see https://symfony.com/doc/current/session.html#making-the-locale-sticky-during-a-user-s-session.
```

Comment:
```suggestion
    The LocaleSwitcher will apply the locale at a request level.
    This means that it will be available only for that request. 
    A redirect, for example, will cancel the LocaleSwitcher's effect.
    For a permanent locale switch between requests see https://symfony.com/doc/current/session.html#making-the-locale-sticky-during-a-user-s-session.
```

About the URL, can you follow the RST format for links please? (see https://github.com/symfony/symfony-docs/pull/20981/files#diff-9ee808b36fb9f3ce98827e47f96d7bf1d53c039fed97fb8364087cd629be779cR1095)